content/_changelogs/10.7.0.md

```diff
@@ -0,0 +1,81 @@
+## 10.7.0
+
+_Released 8/30/2022_
+
+**Features:**
+
+- Re-introduces the experimental
+  [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio).
+  Cypress Studio provides a visual way to generate tests within Cypress by
+  recording interactions against the application under test. Cypress Studio is
+  available only in end-to-end tests, and must be enabled via the
+  `experimentalStudio` flag. Addresses
+  [#23461](https://github.com/cypress-io/cypress/issues/23461)
+- Adds [Svelte](https://svelte.dev/) component testing support. Addresses
```

Comment:
Is this experimental / beta support? Also I assume we should have closed out & referenced the epic for this work? https://github.com/cypress-io/cypress/issues/23037

if the epic isn't correct, I'd update this to reference https://github.com/cypress-io/cypress/pull/23553 that added this code to develop since these issues were tasks to completing the single feature and we don't need to be as granular for the initial release.

Reply:
I just found [this guide](https://docs.google.com/document/d/1pOZ0Ntm4b1Eev_iY5vLBPhOxx66Z8Njrvvg6WMCvWRI/edit#heading=h.ok2apo8hzldd) recently that DX put together, but it calls out ct testing should be capitalized as `Component Testing`. I noticed in a few places.